app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CommentsController < ApplicationController
+	def create
+		@postquote = Postquote.find_by(id: params[:postquote_id])
```

Comment:
Try be consistent with your indentation - use two spaces consistently rather than tabs. You should be able to set sublime to do that.